config/spryk/spryks/Spryker/Common/Test/EnableTestHelper.yml

```diff
@@ -0,0 +1,31 @@
+spryk: update-yml
+description: "Updates the *module* test configuration and adds a test helper."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: Spryker
+
+    layer: ~
+
+    helperClassName:
+        isOptional: true
+
+    afterElement:
+        isOptional: true
+
+    addToElement:
+        value: "suites.{{ layer }}.modules.enabled"
+
+    targetPath:
+        value: "tests/{{ organization }}Test/Zed/{{ module }}/codeception.yml"
+
+    content:
```

Comment:
Why this?

Reply:
To make helperClassName an alias to the content parameter

Reply:
Ok